mozilla/blurts-server
# MNTOR-1181: add breach resolution test

# References: 
Jira: MNTOR-1181
Figma: n/a




# Description
Tests breach resolution:
1. Bypass login with globalSetup
2. Sanity check for data breach page's sections
3.  Prepare for testing: clear all resolved check boxes
4. Click on unresolved breaches tab and resolve the first breach
5. check for the label update for the resolved breach tab: `1 Resolved Breaches`


# Screenshot (if applicable)

Not applicable.

# How to test
Run locally:
1. `npm run e2e -- --headed` to see it in action


# Checklist (Definition of Done)
